### PR TITLE
Rubocop: fix violations of Layout/LineLength

### PR DESCRIPTION
**Pull request by rhymes**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Fixes all violations of `Layout/LineLength` and enables the check for the future (the default is 120 lines which I think it's reasonable).

Most were done automatically, some had to be done manually.

I realize it's a huge PR but it's 99.9% only line length changes 

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

### CODE REVIEW

File: app/controllers/html_variants_controller.rb [hunk 1]
```diff
@@ -3,15 +3,18 @@ class HtmlVariantsController < ApplicationController
...
-                     else
-                       HtmlVariant.where(published: true, approved: true).order("success_rate DESC").includes(:user).page(params[:page]).per(30)
-                     end
+
+    relation = if params[:state] == "mine"
+                 current_user.html_variants
+               elsif params[:state] == "admin"
+                 HtmlVariant.where(published: true, approved: false)
+               elsif params[:state].present?
+                 HtmlVariant.where(published: true, approved: true, group: params[:state])
```
[the remark this answers is not in the document]
Comment (rhymes) [reply app/controllers/html_variants_controller.rb:12]: no. definitely a manual copy paste error! thanks for catching it! I'll see if I can write a test for it

**Comment by rhymes** [comment]
@fdoxyz we should definitely update, great catch! We always forget about Code Climate :D Let me update and see if they go down.

**Comment by rhymes** [comment]
@fdoxyz went down to 4 ahahha

**Comment by fdoxyz** [comment]
Major difference haha! 

I think most or all of these 4 issues can be fixed with a small refactor now, right? That way we'll end up with a clean slate on codeclimate's rubocop

**Comment by rhymes** [comment]
@fdoxyz I think they are out of scope for this PR though. We might want to deal with them individually as cyclomatic complexity means that the method does too many things

**Pull request merged**